### ISSUE TITLE
Stripe payment glitches

### ISSUE THREAD

**Issue by StuartMcD** [solspace/craft3-freeform#1559]
### What happened?

So have Stripe/payments working and making test payments with Stripe using dev mode detail. Trying to display a receipt style page where I use:

```
{% set payment = freeform.submissions({token: paymentToken}).one() %}{# The FF payment record #}
{% set payments = freeform.payments(payment.id) %} {# The Stripe record #}
...
<p>A receipt has been sent to: {{payment.email}}.  ${{payments.amount}} {{payments.currency}} {{payments.card}} | {{payments.id}}</p>
```
**Email**: renders fine
!! **payments.amount**: renders only the whole dollars (no cents) Displayed in admin with cents !!
**payments.currency**: renders fine
**payments.card**: renders fine
**payments.id**: does not render (throws error on dev site - see below)

Also tested and **not** working with similar errors:
payments.gateway
payments.resourceId

### Errors and Stack Trace (if available)

```shell
Calling unknown method: Solspace\Freeform\Models\Payments\PaymentModel::id()
```

### How can we reproduce this?

1. 
2. 
3. 

### Freeform Edition

Pro

### Freeform Version

5.6.4

### Craft Version

5.4.4

**Comment by gustavs-gutmanis** [comment]
Thank you for reporting these @StuartMcD, we will release a fix for this in the upcoming version.

**Comment by kjmartens** [comment]
This is resolved in Freeform 5.6.6+ now. 🙂 

However, `payments.gateway` is not available at this time.
